numpy/core/tests/test_umath_complex.py

```diff
@@ -134,8 +134,8 @@ def test_simple(self):
         x = np.array([1+0j, 1+2j])
         y_r = np.log(np.abs(x)) + 1j * np.angle(x)
         y = np.log(x)
-        for i in range(len(x)):
-            assert_almost_equal(y[i], y_r[i])
+        for yi, y_ri in zip(y, y_r):
+            assert_almost_equal(yi, y_ri)
```

Comment:
Couldn't this be written more compactly without the loop as `assert_almost_equal(y, y_r)`? What am I missing?

Reply:
Must have been an oversight, as `assert_almost_equal` takes array_like args, and seems to work with complex types. Done 3x.